crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -430,4 +474,160 @@ mod test {
         let asset = Denom::from("other_port/source_channel/asset".to_string());
         assert!(!is_prefixed(&source_port, &source_channel, &asset));
     }
+
+    #[tokio::test]
+    async fn execute_ics20_transfer_to_eoa() {
```

Comment:
What does `to_eoa` mean? What does this test?

Reply:
to externally owned account, so not a bridge account. this is testing an IBC transfer coming in to a non-bridge account

Reply:
Can you `s/eoa/externally_owned_account`? 🙏